messaging/kafka-avro-reactive-messaging/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.ts.openshift</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <artifactId>kafka-avro-reactive-messaging</artifactId>
+
+    <name>Quarkus OpenShift TS: Messaging: Reactive Kafka with Avro</name>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-openshift</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus.ts.openshift</groupId>
+            <artifactId>app-metadata</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-reactive-messaging-kafka</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-health</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jsonb</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-avro</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-scheduler</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.apicurio</groupId>
+            <artifactId>apicurio-registry-utils-serde</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jboss.spec.javax.interceptor</groupId>
+                    <artifactId>jboss-interceptors-api_1.2_spec</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <!-- quarkiverse-apicurio-registry-client: native mode dependency -->
+        <dependency>
+            <groupId>io.quarkiverse.apicurio</groupId>
+            <artifactId>quarkiverse-apicurio-registry-client</artifactId>
```

Comment:
I don't feel competent enough to review this change because as far I know, quarkiverse is a community thing and this needs to work using product Quarkus. 
I'm aware that this is a workaround to make it work in Native mode, but I'm missing some links to the issues where the final fix should be included (regarding to documentation and/or a core extension).
cc @rsvoboda

Reply:
We've talked with @famartinrh , and request him (to his team) to change the artifact name to something like `quarkiverse-apicurio-registry-native-client` in order to describe better the goal of this extension. Also, we suggested to push this dependency to [Quarkus maven bom ](https://repo1.maven.org/maven2/io/quarkus/quarkus-universe-bom/1.10.5.Final/quarkus-universe-bom-1.10.5.Final.pom)

Reply:
So, with this change and when running the suite using the product artifacts, will the test fail?